responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -29,12 +29,19 @@
 _TARGET_COLUMN = 'target_column'
 _TASK_TYPE = 'task_type'
 _CLASSES = 'classes'
+_COLUMNS = 'columns'
+_FEATURERANGES = 'feature_ranges'
```

Comment:
nit: maybe add an underscore "_" here, similar to variable value, so _FEATURE_RANGES

Reply:
thanks, updated